x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_landing.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiButton,
+  EuiLink,
+  EuiPageHeader,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { getAppSearchUrl } from '../../../shared/enterprise_search_url';
+import { ENGINE_CRAWLER_PATH } from '../../routes';
+import { generateEnginePath } from '../engine';
+
+import './crawler_landing.scss';
+import { CRAWLER_TITLE } from '.';
+
+export const CrawlerLanding: React.FC = () => (
+  <div data-test-subj="CrawlerLanding" className="crawler-landing">
+    <EuiPageHeader pageTitle={CRAWLER_TITLE} />
+    <EuiSpacer />
+    <EuiPanel grow paddingSize="l" className="crawler-landing__panel">
+      <div>
+        <EuiTitle size="s">
+          <h1>
+            {i18n.translate('xpack.enterpriseSearch.appSearch.engine.crawler.landingPage.title', {
+              defaultMessage: 'Setup the Web Crawler',
+            })}
```

Comment:
The more I do these inline `i18n.translate`s in React the more I feel like I should be using the react component version.

Reply:
Ahhhhh I always forget about the testing angle, that is very frustrating

Reply:
Ya for sure, otherwise I'd be down for switching for sure. Always gotta be either types or tests 😅